docker-compose.yml

```diff
@@ -45,8 +45,8 @@ services:
       - EZUNPAYWALL_NOTIFICATIONS_RECEIVERS
       - EZUNPAYWALL_NOTIFICATIONS_MACHINE
        # unpaywall
-      - API_KEY_UPW
-      - UNPAYWALL_URL=http://api.unpaywall.org/feed/changefiles
+      - UNPAYWALL_APIKEY
+      - UNPAYWALL_HOST=http://api.unpaywall.org
```

Comment:
Why changing from `URL` to `HOST` ? `http://api.unpaywall.org` looks like a URL (where `api.unpaywall.org` is the host)